src/components/widget/WidgetRecommendedBy.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+  <div class="widget widget--recommended-by">
+    <div class="widget__header d-flex align-items-center" :class="{ 'card-body': widget.card }">
+      <fa icon="users" fixed-width class="mr-2" />
+      <h3 class="m-0 p-0 h5">Latest recommended documents</h3>
+    </div>
+    <v-wait :for="loader" transition="fade">
+      <div slot="waiting" class="widget__spinner text-center p-4">
+        <fa icon="circle-notch" spin size="2x"></fa>
+      </div>
+      <div class="list-group widget__list" :class="{ 'list-group-flush': widget.card }">
+        <a
+          v-for="({ document, href, user, creationDate }, i) in items"
+          :key="i"
+          class="list-group-item list-group-item-action widget__list__item d-flex align-items-center flex-truncate"
+          :href="href"
+        >
+          <div v-if="!widget.hideThumbnails && widget.cols >= 6" class="d-none d-md-block flex-shrink-0">
+            <document-thumbnail :document="document" crop lazy class="mr-3" />
+          </div>
+          <div class="flex-grow-1">
+            <div class="widget__list__item__label d-flex align-items-start">
+              <document-sliced-name wrap :document="document" class="text-primary" />
+              <div class="ml-auto pl-1 d-flex align-items-center text-muted flex-shrink-0">
+                <user-display :username="user.id" hide-link class="text-nowrap" />
+                <span
+                  v-if="creationDate"
+                  v-b-tooltip
+                  class="ml-1 text-nowrap"
+                  :title="creationDate | humanLongDate($i18n.locale)"
```

Comment:
Since filters will be deprecated in Vue 3 I would suggest to use computed properties instead (but I also like this feature so I don't mind if stay as is).

Reply:
We still use the filter in many places so I guess we can refactor them all at once!